mirascope/core/bedrock/_utils/_calculate_cost.py

```diff
@@ -0,0 +1,8 @@
+"""Calculate the cost of a completion using the Bedrock API."""
+
+
+def calculate_cost(
+    input_tokens: int | float | None, output_tokens: int | float | None, model: str
+) -> float | None:
+    """Calculate the cost of a completion using the Bedrock API."""
+    return None
```

Comment:
can you please add a comment here mentioning that we're looking into a way to determine the cost dynamically since there are too many to support for this initial PR?